core-processor/src/common.rs

```diff
@@ -195,6 +198,13 @@ pub enum JournalNote {
         /// Value amount
         value: u128,
     },
+    /// Store programs requested by user to be initialized later
+    StoreNewPrograms {
+        /// Code hash used to create new programs with ids in `candidates` field
```

Comment:
It is our overall problem. Doc-comments are to be ended with a dot (see `std` docs).

Reply:
done